coap-gateway/config.yaml

```diff
@@ -24,42 +24,42 @@ apis:
       ownerClaim: "sub"
       deviceIdClaim: ""
       providers:
-      - name: "plgd.web"
-        clientID: ""
-        clientSecretFile: ""
-        scopes: []
-        audience: ""
-        redirectURL: "https://localhost/things"
-        authority: ""
-        http:
-          maxIdleConns: 16
-          maxConnsPerHost: 32
-          maxIdleConnsPerHost: 16
-          idleConnTimeout: 30s
-          timeout: 10s
-          tls:
-            caPool: "/secrets/public/rootca.crt"
-            keyFile: "/secrets/private/cert.key"
-            certFile: "/secrets/public/cert.crt"
-            useSystemCAPool: false
-      - name: "plgd.mobile"
-        clientID: ""
-        clientSecretFile: ""
-        scopes: []
-        audience: ""
-        redirectURL: "cloud.plgd.mobile://login-callback"
-        authority: ""
-        http:
-          maxIdleConns: 16
-          maxConnsPerHost: 32
-          maxIdleConnsPerHost: 16
-          idleConnTimeout: 30s
-          timeout: 10s
-          tls:
-            caPool: "/secrets/public/rootca.crt"
-            keyFile: "/secrets/private/cert.key"
-            certFile: "/secrets/public/cert.crt"
-            useSystemCAPool: false
+        - name: "plgd.web"
```

Comment:
python yaml parser warned you about this indentation?

Reply:
I have automatic formatting on save enabled in Visual Studio Code, so it go reformatting automatically as I've changed something and saved.